apepay/__init__.py

```diff
@@ -58,9 +58,44 @@ def normalize_address(cls, value: Any) -> AddressType:
 
     @property
     def contract(self) -> ContractInstance:
-        # TODO: Fix this
-        return self.project_manager.StreamManager.at(self.address)
-        # return self.chain_manager.contracts.instance_at(self.address)
+        try:
+            return self.chain_manager.contracts.instance_at(self.address)
+        except Exception:
```

Comment:
what types of exceptions do you expect here? wondering if we could mitigate anything upstream?

Reply:
Could reduce to the "contract not found" error, only need to fallback on a local network or if we don't have an explorer (although it still won't work unless we somehow bundle the manifest)

Reply:
- [ ] Release ape 0.6.14 and bump here